wasm-engines/docker/Dockerfile

```diff
@@ -195,13 +195,34 @@ RUN update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 1
 RUN git clone https://github.com/wasmx/fizzy.git --single-branch --branch master
 RUN cd fizzy && mkdir build && cd build && cmake -DFIZZY_TESTING=ON .. && cmake --build .
 
+FROM ubuntu:19.10
+
+# install python 2.7
+RUN apt update && apt-get install -y wget python2.7 curl
+
+# install python 3.7
+RUN apt-get install -y python3.7
+RUN apt-get install -y python3-distutils
```

Comment:
Definitely merge these two into a line. Try to merge `RUN` commands as they create checkpoints.